src/osd/ReplicatedBackend.cc

```diff
@@ -1981,20 +1981,25 @@ int ReplicatedBackend::build_push_op(const ObjectRecoveryInfo &recovery_info,
   }
 
   uint64_t available = cct->_conf->osd_recovery_max_chunk;
+  uint64_t omap_available = cct->_conf->osd_recovery_max_omap_entries_per_chunk;
   if (!progress.omap_complete) {
+    uint64_t size;
```

Comment:
Why introduce this uninitialized here?  Move this declaration into the for loop where it is initialized.